yt_dlp/extractor/streamtape.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import js_to_json, urljoin
+
+# strings are obfuscated by concatenating substrings
+split_string_part = r'(?:%s|%s)' % (r'"(?:[^"\\]|\\.)*"',
+                                    r"'(?:[^'\\]|\\.)*'")
+split_string = r'(?:' + split_string_part + r'(?:\s*\+\s*' + split_string_part + r')*)'
+videolink = r"(?:'\+')?".join('videolink')
+videolink = r"document\.getElementById\('" + videolink + r"'\)\.innerHTML\s*=\s*(?P<data>" + split_string + r")"
+
+
+class StreamtapeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?streamtape.com/[ev]/(?P<id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://streamtape.com/v/AJD9gAVwMOcXrQ8/Big_Buck_Bunny_Trailer_400p.ogg',
```

Comment:
This URL no longer exists. Write a new test

Reply:
Wrote new test